apis/kubedb/v1alpha2/perconaxtradb_types.go

```diff
@@ -126,13 +126,22 @@ type PerconaXtraDBSpec struct {
 	SystemUserSecrets *SystemUserSecretsSpec `json:"systemUserSecrets,omitempty"`
 }
 
-// +kubebuilder:validation:Enum=server;client;metrics-exporter
+// +kubebuilder:validation:Enum=server;client;exporter
 type PerconaXtraDBCertificateAlias string
 
 const (
-	PerconaXtraDBServerCert          PerconaXtraDBCertificateAlias = "server"
-	PerconaXtraDBClientCert          PerconaXtraDBCertificateAlias = "client"
-	PerconaXtraDBMetricsExporterCert PerconaXtraDBCertificateAlias = "metrics-exporter"
+	PerconaXtraDBServerCert   PerconaXtraDBCertificateAlias = "server"
+	PerconaXtraDBClientCert   PerconaXtraDBCertificateAlias = "client"
+	PerconaXtraDBExporterCert PerconaXtraDBCertificateAlias = "exporter"
```

Comment:
Why are we changing metrics-exporter -> exporter?

Reply:
Made it same as the certificate mount path `/etc/mysql/certs/exporter`